### DISCUSSION TITLE
使用clash代理时报错

### DISCUSSION THREAD

**Discussion by gwongsam** [ChatGPTNextWeb/NextChat#1310] [Q&A]
这到底是哪里出问题了，我在终端测试代理已经成功开启了，目标平台Ubuntu，代理工具clash
1、启动方式如下
`docker run -d -p 3000:3000 \
   --name http_normal \
   -e OPENAI_API_KEY="" \
   -e CODE="" \
   -e PROXY_URL="http: //127.0.0.1:7890" \
   yidadaa/chatgpt-next-web`
2、报错信息如下
`{
  "cause": {
    "errno": -111,
    "code": "ECONNREFUSED",
    "syscall": "connect",
    "address": "224.0.0.1",
    "port": 443
  }
}`

**Comment by Yidadaa** [comment]
http: 后面多了个空格

**Reply by gwongsam** [reply]
很少用docker不太熟悉，后面发现是不能使用localhost，因为我的代理是在宿主机上配置的，使用localhost只能连接到容器自身localhost，改为使用宿主机局域网地址就行了

**Reply by Liwenhao-426** [reply]
同样的问题，想请教下如何更改宿主机局域网地址？是在网络那host、bridge里设置还是在执行命令里更改
